### DISCUSSION TITLE
I hope the manager can have a look at how I feel when I play godot these days

### DISCUSSION THREAD

**Discussion by xiaoliang6669** [godotengine/godot-proposals#5933] [General Discussions]
I feel that some maintainers are too arrogant. Any submitted bug will make you upload the project. If you don't upload it, your submission will be closed. As if they were high above the ground, I just play around and mention bugs for the sake of godot. But I am not obliged to write projects for you to analyze. Besides, I am also a beginner who has to work hard to write projects. It doesn't matter whether you adopt my questions or not. I don't ask you to put your position in order. This kind of ethos does not change, and you can play with godot by yourself.

**Comment by xiaoliang6669** [comment]
Please put yourself in the right place. At present, many people who play godot are at the learning stage. They are very hard to write projects. No one will rely on godot, nor expect you to solve problems. If you are not happy, just change the engine. Don't be too arrogant

**Comment by Spartan322** [comment]
A minimal project exists to confirm that errors and bugs can be confirmed easily by contributors, (its not like anyone here is trying to analyze what you do beyond needing to understand why its breaking the engine in unreasonable manners) this is necessary to QA the engine and also to confirm the fix actually worked consistently, popular projects already have a wide spread problem of misreported bugs, or reporting of fixed bugs, or bug reports that confuse the core issue, its got nothing to do with arrogance, its about having a scalable process of handling issues for the maintainers and contributors so things can actually be fixed, nobody can even confirm if its specific to your system or setup or that it isn't a bug in your system without this. Godot is not so difficult that if you're working with the engine that you shouldn't be able to provide a minimal project for issue replication, how were you able to come across the issue in the first place than? If you can't figure out a manner to create a simple replication project that replicates it there are multiple community endeavors you can take, like the discord, forums, or rocket chat, for which you can get quick advice on either what's going wrong or how to get it fixed. And if you can't even somewhat consistently repeat the issue in a simple manner, what chance will any of us ever be able to fix the problem? Or either confirm absolutely and definitely it was fixed?

**Comment by xiaoliang6669** [comment]
forget it. Are our users your employees? For some problems, it is clear that you only need to type two sentences of code to see whether my feedback is correct or not, and the maintainer is like a robot. Every feedback allows you to provide the smallest project to upload

**Reply by Spartan322** [reply]
None of this addresses what was said, if you needed help setting up a MRP and are unsure of yourself doing so, you could easily ask someone to help, you have plenty of chances to actually give reasonable feedback even if you don't know how by asking, if you are a beginner than in being humble we'll gladly try to assist into making you become better. It is also very possible as well for a bug report to be from a misunderstanding of the engine as well and caused by user error, maybe it could add to additional clarity needed for the engine from which the user error spawned, but again without an MRP there is no way to even confirm this and actually help you that way either. Providing generic code and steps to achieve it are not enough to ensure a consistent test that validates whether an issue is an issue and whether it has been fixed. And none of the the maintainers are like robots, they're merely being reasonable and expecting each individual that opens an issue to put some time into assisting in whatever way they can so the engine can improve.

**Comment by xiaoliang6669** [comment]
stop here, I'm fed up with you defenders. I will never give any feedback any more. This behavior makes me feel like a dominated clown

**Comment by AThousandShips** [comment]
You are the one that expect *us* to do a lot of extra work for free

You have the issue happening to you, because of this you can see when it happens, in what situation it happens, so it's much easier for you to make an example of it, so you can spend a little time doing that, being helpful, or you can demand that people spend a lot of extra time finding what is wrong with only your vague description. If anyone is arrogant it is you honestly.
